java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java

```diff
@@ -46,17 +47,16 @@ public static void main(String[] args) {
         for (ClientName client : runConfiguration.clients) {
             switch (client) {
                 case JEDIS:
-                    // run testClientSetGet on JEDIS sync client
                     System.out.println("Run JEDIS sync client");
                     testClientSetGet(JedisClient::new, runConfiguration, false);
                     break;
                 case LETTUCE:
-                    // run testClientSetGet on LETTUCE async client
                     System.out.println("Run LETTUCE async client");
                     testClientSetGet(LettuceAsyncClient::new, runConfiguration, true);
                     break;
                 case GLIDE:
-                    System.out.println("GLIDE for Redis async not yet configured");
+                    System.out.println("Run GLIDE async client");
```

Comment:
woohoo
does it run in CI?

Reply:
No.  Does it need to?

Reply:
https://github.com/Bit-Quill/glide-for-redis/actions/runs/7841055895
But it didn't trigger benchmarking.

Reply:
Oh you need to add protobuf installation to benchmarking CI
Please update it to let CI pass and generate numbers.